includes/Order/functions.php

```diff
@@ -49,10 +49,12 @@ function dokan_get_seller_amount_from_order( $order_id, $get_array = false ) {
  * @param int $limit
  * @param int $offset
  * @param int $customer_id
+ * @param int $total_high
+ * @param int $total_low
  *
  * @return array
  */
-function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = null, $limit = 10, $offset = 0, $customer_id = null ) {
+function dokan_get_seller_orders( $seller_id, $status = 'all', $order_date = null, $limit = 10, $offset = 0, $customer_id = null, $total_high = '', $total_low = '', $start_date = '', $end_date = '' ) {
```

Comment:
Making so many distinct parameters isn't the proper way of doing it. If possible, keep the `$seller_id` param, and move other params inside a `$args` array.